lib/Doctrine/DBAL/Schema/AbstractSchemaManager.php

```diff
@@ -423,7 +423,7 @@ public function dropView($name)
      *
      * @param string $database The name of the database to create.
      *
-     * @return void
+     * @return void|bool
```

Comment:
Looks like a bug. Mixing something with void doesn't make sense.

Reply:
Yep, removed. But `OracleSchemaManager::createDatabase()` returns true so I removed that one. Should be documented in UPGRADING?